packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -28,11 +28,12 @@ import {
 	waitForContainerConnection,
 } from "@fluidframework/test-utils";
 import { describeNoCompat, itExpects } from "@fluid-internal/test-version-utils";
-import { ConnectionState } from "@fluidframework/container-loader";
+import { ConnectionState, IPendingContainerState } from "@fluidframework/container-loader";
```

Comment:
@dannimad `IPendingContainerState` has been removed on `next`. Any recommendations for how to proceed with the merge for this?

Reply:
I went ahead and just used `any` instead for the time being, since this is test code. But we can always change to something else if there is a preferable alternative.